x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_events.tsx

```diff
@@ -68,7 +68,7 @@ export const AnalyticsCollectionEvents: React.FC<AnalyticsCollectionEventsProps>
           field: 'labels.user_uuid',
         },
       ]}
-      query={`_index: logs-elastic_analytics.events-${collection.name}*`}
+      query={`_index: ${collection.events_datastream}`}
```

Comment:
The query was broken and the event view was not working.
Now using the data stream directly